### PR TITLE
Extra earrings xp

### PR DESCRIPTION
**Pull request by weffjebster**
added setting to add extra xp to galaxy earrings bonus
values 0-24
additional means on top of the 1xp per orb per kill
if you want galaxy earrings to give 10xp per orb per kill you should select 9

### CODE REVIEW

File: TsRandomizer/Randomisation/OrbExperienceManager.cs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+		{
+			var inventory = level.GameSave.Inventory;
+			var hasEarringsEquipped = inventory.EquippedTrinketA == EInventoryEquipmentType.NelisteEarring
+					|| inventory.EquippedTrinketB == EInventoryEquipmentType.NelisteEarring;
+
+			if (!hasEarringsEquipped) return;
+			// minus the xp from the base game, minus the xp from the call to GiveOrbExperience to run the level up logic
+			int extraXpToAdd = (int)(extraXp - 2);
+			var levelReflected = level.AsDynamic();
+			dynamic orbManager = lunais.AsDynamic()._orbManager;
```
Comment (Jarno458) [inline TsRandomizer/Randomisation/OrbExperienceManager.cs:66]: plz dont do multile calls to `lunais.AsDynamic()` instead make it variable

**Review by Jarno458 — CHANGES_REQUESTED** [review]

File: TsRandomizer/Randomisation/OrbExperienceManager.cs (hunk 1 above)
Comment (weffjebster) [reply TsRandomizer/Randomisation/OrbExperienceManager.cs:66]: One of these is level, not lunais. Although i can pass in the exact property i need from level reflected from levelobject.cs so I'll do that

Comment (weffjebster) [reply TsRandomizer/Randomisation/OrbExperienceManager.cs:66]: Oh. Github on my phone didn't show me the proper code. Yeah I'll fix this

Comment (weffjebster) [reply TsRandomizer/Randomisation/OrbExperienceManager.cs:66]: i have done the fixing

**Review by Jarno458 — APPROVED** [review]

**Pull request closed without merging**